src/main/java/domain/number/LottoNumberGenerator.java

```diff
@@ -0,0 +1,12 @@
+package domain.number;
+
+import java.util.List;
+
+public interface LottoNumberGenerator {
+    int MIN_NUMBER = 1;
+    int MAX_NUMBER = 45;
+    int LOTTO_BALLS_IN_LOTTO = 6;
+
+    List<List<Integer>> createLottoNumberBundle(final int quantity);
+    List<Integer> createLottoNumber();
```

Comment:
```suggestion
    List<Integer> create();
```

`LottoNumberGenerator.createLottoNumber()` vs `LottoNumberGenerator.create()`

어떠한 차이가 있을지 고민해보아요

Reply:
이미 클래스의 이름이 RandomLottoNumberGenerator이니, 굳이 메서드명에서도 createLottoNumber라는 네이밍을 해줄 필요가 없을 것 같네요. 

createLottoNumberBundle을 createBundle로, createLottoNumber를 create로 바꿨습니다.